ci/ReleaseTool/Common.cs

```diff
@@ -162,5 +303,52 @@ public static (string, int) ExtractPullRequestInfo(string pullRequestUrl)
 
             return (repoName, pullRequestId);
         }
+
+        public static string GetReleaseNotesFromChangeLog(NLog.Logger Logger)
+        {
+            if (!File.Exists(ChangeLogFilename))
+            {
+                throw new InvalidOperationException("Could not get draft release notes, as the change log file, " +
+                    $"{ChangeLogFilename}, does not exist.");
+            }
+
+            Logger.Info("Reading {0}...", ChangeLogFilename);
+
+            var releaseBody = new StringBuilder();
+            var changedSection = 0;
+
+            using (var reader = new StreamReader(ChangeLogFilename))
+            {
+                while (!reader.EndOfStream)
+                {
+                    // Here we target the second Heading2 ("##") section.
+                    // The first section will be the "Unreleased" section. The second will be the correct release notes.
+                    var line = reader.ReadLine();
+                    if (line.StartsWith("## "))
+                    {
+                        changedSection += 1;
+
+                        if (changedSection == 3)
```

Comment:
Hm, I know this is copied, but I think we can move the break to after the `releaseBody.AppendLine`